kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -417,6 +448,11 @@
         return this.usersPins.map(pin => pin.instance_id);
       },
     },
+    provide() {
+      return {
+        $layoutSpan: () => this.layoutSpan,
```

Comment:
Heads up that this is a [work around solution](https://github.com/vuejs/vue/issues/7017#issuecomment-480906691) to using computed properties specified [here](https://vuejs.org/guide/components/provide-inject.html#working-with-reactivity) as adding the configuration failed to work

Reply:
Do you think that maybe to avoid the workaround this could be a `computed()` property in some kind of `useLayout` composable so we can define the `layoutSpan` calculation once? I wonder if there are other places we could simplify with such a composable to have things like "channelCardLayoutSpan", "contentCardLayoutSpan".

Anyway, just a thought for the future, no changes needed in this regard now.